modules/ROOT/pages/notifications.adoc

```diff
@@ -0,0 +1,19 @@
+= Important Notifications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Use these sources to stay informed about new releases and downtime for Anypoint API Community Manager.
+
+== Release Announcements
+
+Keeping the API Community Manager components updated is critical to keep your API community operating smoothly and optimally. Watch for new releases in these channels so you can plan to xref:update-acm.adoc[update your installation] at your convenience.
+
+New releases are announced at http://trust.mulesoft.com in the "Scheduled Maintenance" section and in xref:release-notes::api-community-manager/api-community-manager-release-notes.adoc[the API Community Manager release notes].
```

Comment:
Consider merging with the previous or creating a two-item list.

Reply:
agree

Reply:
Fixed.